### PR TITLE
enh(ci): MON-33464 Keep package managers in centreon web docker slim images

### PR DESCRIPTION
**Pull request by kabreu**
Modify the list of binaries and path to keep in the docker slim images so that package managers can be used (apt/dpkg, dnf/rpm)

### CODE REVIEW

File: .github/docker/centreon-web/alma8/slim-configuration/include-bin.txt [hunk 1]
```diff
@@ -5,3 +5,12 @@
...
 /usr/lib64/libldap-2.4.so.2
+/usr/lib64/librpm.so.8
+/usr/bin/dnf
+/usr/lib64/libstdc++.so.6
+/usr/lib64/libdnf.so.2
+/usr/lib64/librpmbuild.so.8
+/usr/lib64/librpmsign.so.8
+/usr/lib64/libcomps.so.0
+/usr/lib64/libncursesw.so.6
+/usr/libexec/platform-python
```
Comment (kduret) [inline .github/docker/centreon-web/alma8/slim-configuration/include-bin.txt:16]:
```suggestion
/usr/bin/rpm
/usr/bin/python3.6
/usr/bin/python3
```

File: .github/docker/centreon-web/alma9/slim-configuration/include-path.txt [hunk 2]
```diff
@@ -36,3 +36,6 @@
 /var/lib/mysql/centreon_storage
 /usr/lib64/nagios/plugins
 /usr/lib/centreon/plugins
+/usr/lib/rpm
+/usr/lib/python3.9
+/usr/lib64/python3.9
```
Comment (kduret) [inline .github/docker/centreon-web/alma9/slim-configuration/include-path.txt:41]:
```suggestion
/usr/bin/dnf
/usr/lib/rpm
/usr/lib/python3.9
/usr/lib64/python3.9
/etc/yum.repos.d
```

File: .github/docker/centreon-web/alma8/slim-configuration/exec.txt [hunk 3]
```diff
@@ -22,4 +22,7 @@ service mysql stop
 
 rm -f /tmp/gorgone/*.ipc /var/log/centreon*/*.log /var/lib/centreon-gorgone/history.sdb
 
+dnf install -y vim
```
Comment (tuntoja) [inline .github/docker/centreon-web/alma8/slim-configuration/exec.txt:25]: is vim mandatory (it adds some weight to the container if not)

Comment (kduret) [reply .github/docker/centreon-web/alma8/slim-configuration/exec.txt:25]: this was mainly for testing purpose in order to allow slimtoolkit to detect binary usage

File: centreon/www/main.php [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (kduret) [inline centreon/www/main.php:2]:
```suggestion
```

**Review by kduret — APPROVED** [review]

**Review by tuntoja — DISMISSED** [review]

**Pull request merged**